contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
-
-### 3. It's open source
-
-Our MIT License isn’t just for show. You can access [our source code](https://github.com/PostHog/posthog), raise your own issues and PRs, and use it to [build your own apps](/docs/apps/build) or even add extra functionality. You also benefit from the work of other teams who build their own apps. And we're not just an open-source tool; we're an open-source company. Our [company handbook](/handbook) is open to everyone, as is [how we pay people](/handbook/people/compensation).
-
-> **Further reading:** [How Amplitude compares to alternatives](/blog/best-amplitude-alternatives)
-
-## Feature comparison
-
-This table compares the Amplitude Analytics 'Growth' plan to PostHog Cloud, our fully-managed SaaS offering.
-
-<ComparisonTable column1="Amplitude Analytics" column2="PostHog">
-  <ComparisonRow column1={false} column2={true} feature="Transparent pricing" />
-  <ComparisonRow column1={false} column2={true} feature="Self-serve" />
-  <ComparisonRow column1={true} column2={true} feature="Multi-site" />
-  <ComparisonRow column1={false} column2={true} feature="Event autocapture" />
-  <ComparisonRow column1={true} column2={true} feature="Funnel analysis" />
-  <ComparisonRow column1={true} column2={true} feature="Cohort analysis" />
-  <ComparisonRow column1={true} column2={true} feature="Conversion tracking" />
-  <ComparisonRow column1={true} column2={true} feature="Retention tracking" />
-  <ComparisonRow column1={true} column2={true} feature="Revenue tracking" />
-  <ComparisonRow column1={true} column2={true} feature="UTM tracking" />
-  <ComparisonRow column1={true} column2={true} feature="Event tracking" />
-  <ComparisonRow column1={true} column2={true} feature="Feature flags" />
-  <ComparisonRow column1={false} column2={true} feature="Heatmaps" />
-  <ComparisonRow column1={false} column2={true} feature="Session replay" />
-  <ComparisonRow column1={"Add-on"} column2={true} feature="Experimentation" />
-  <ComparisonRow column1={true} column2={true} feature="Predictive analytics" />
-  <ComparisonRow column1={false} column2={true} feature="User surveys" />
-  <ComparisonRow column1={false} column2={true} feature="Hedgehogs" />
+PostHog autocaptures data, so you don't need to waste energy instrumenting basic events. You can choose between light and dark mode themes. Features like our custom SQL query builder, keyboard shortcuts, and command palette, cater to power users. You can setup PostHog and start paying us without ever speaking to a sales rep, though our [customer success team](/contact-sales) is always available to answer your questions.
```

Comment:
Autocapture and dark mode seems like something non-technical people would appreciate, rather than engineers.

Engineers would care more about SDKs, simple installation for multiple integrated tools, API, HogQL, EU hosting.